libs/application/template-api-modules/src/lib/modules/templates/institution-collaboration/emailGenerators/applicationEmail.ts

```diff
@@ -44,13 +44,19 @@ export const generateApplicationEmail: ApplicationEmail = (
       )
     : []
 
+  const contactName = getValueViaPath(application.answers, 'contact.name') as
+    | string
+    | undefined
+  const contactEmail = getValueViaPath(application.answers, 'contact.email') as
+    | string
+    | undefined
   const overview = applicationOverviewTemplate(application)
   const body = dedent(`<h2>Yfirlit umsóknar</h2> ${overview}`)
 
   return {
     from: {
-      name: applicationSenderName,
-      address: applicationSenderEmail,
+      name: contactName ?? applicationSenderName,
+      address: contactEmail ?? applicationSenderEmail,
```

Comment:
Hm, what's the use case for this? I assume spam filters will block us if we start sending emails from arbitrary email addresses.

Reply:
Good point, I'll raise the issue with our PO

Reply:
The usecase is that there is a ticketing system on the other end (Zendesk) where communication with the applicant takes place. In the current setup, the sender is island@island.is so notifications from the ticketing system never reach the applicant.
That being said, we do indeed risk being flagged as SPAM if we introduce this functionality.
I'm discussing this with our BO

Reply:
Update. Lets scrap this functionality.
We will take a look at using an API with the ticketing system after the summer.

Reply:
@atligudl FYI